ft_databrowser.m

```diff
@@ -658,6 +663,16 @@
 % open a new figure with the specified settings
 h = open_figure(keepfields(cfg, {'figure', 'position', 'visible', 'renderer'}));
 
+% check colormap is in proper format and set it
+if isfield(cfg, 'colormap')
+  if ischar(cfg.colormap)
+    cfg.colormap = ft_colormap(cfg.colormap);
+  elseif iscell(cfg.colormap)
+    cfg.colormap = ft_colormap(cfg.colormap{:});
```

Comment:
so this allows to pass the number of steps, right? I see that this is also how it is done in topoplot_common. Can you check that all uses of ft_colormap work consistently? That also means that the check

```
  if size(cfg.colormap,2)~=3
    ft_error('cfg.colormap must be Nx3');
  end
```

from topoplot_common should be added here.

Reply:
OK, done. The ft_topo<something> rely on topoplot_common, so no action needed there. I addressed it in the relevant multi<> and single<>